packages/block-editor-utils/src/controls/Checkbox.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { CheckboxControl } from '@wordpress/components';
+import { ControlProps } from '../types/index.js';
+
+function Checkbox<T extends Record<string, any>>({
+  config,
+  props,
+}: ControlProps<T>) {
+  const onChange = (newContent: boolean) => {
+    props.setAttributes({ [config.name]: newContent });
+  };
+  return (
+    // TODO: ensure contents match CheckControl in Block Editor- what is help value?
```

Comment:
Based on @theodesp answers, you can remove this TODO.

Reply:
Thank you, will do!